build.sbt

```diff
@@ -468,13 +717,14 @@ lazy val bench = http4sProject("bench")
   .settings(
     description := "Benchmarks for http4s",
     startYear := Some(2015),
-    libraryDependencies += circeParser,
+    libraryDependencies += circeParser.value,
     undeclaredCompileDependenciesTest := {},
-    unusedCompileDependenciesTest := {},
+    unusedCompileDependenciesTest := {}
   )
   .dependsOn(core, circe)
 
-lazy val docs = http4sProject("docs")
+// Workaround via full cross
+lazy val docs = http4sProject("docs", CrossType.Full, List(JVMPlatform))
```

Comment:
I don't think I care about this CrossType.Full, but I'd still like to understand it.

Reply:
Almost definitely the same issue as above. Or, even more weirdness :(